docs/index.html

```diff
@@ -20,7 +20,7 @@
     <header class="header">
```

Comment:
I don't know how to comment on the above BIN file.  What is the "docs/docsets/Workflow.tgz" file used for?

Reply:
No clue, jazzy makes a thing, I check in the thing jazzy creates.